config-templates/redhat/odf-remote/4.9/custom-parameters.json

```diff
@@ -82,6 +96,10 @@
     "displayname": "Encryption enabled",
     "name": "cluster-encryption",
     "default": "false",
+    "regex": "^(?:tru|fals)e$",
```

Comment:
same here `"regex": "^(?:true|false)$",`

Reply:
Addressed in https://github.com/IBM/ibm-satellite-storage/pull/194/commits/bfad1599711afef7153182a023ded18ea5e91331